traffic_ops/app/db/seeds.sql

```diff
@@ -659,6 +666,10 @@ insert into api_capability (http_method, route, capability) values ('PUT', 'serv
 insert into api_capability (http_method, route, capability) values ('POST', 'servers/*/update_status', 'servers-write') ON CONFLICT (http_method, route, capability) DO NOTHING;
 insert into api_capability (http_method, route, capability) values ('GET', 'servers/checks', 'servers-read') ON CONFLICT (http_method, route, capability) DO NOTHING;
 insert into api_capability (http_method, route, capability) values ('POST', 'servercheck', 'servers-write') ON CONFLICT (http_method, route, capability) DO NOTHING;
+-- server capabilities
+insert into api_capability (http_method, route, capability) values ('GET', 'server_server_capabilities', 'server-server-capabilities-read') ON CONFLICT (http_method, route, capability) DO NOTHING;
```

Comment:
I wonder if it makes more sense to add these endpoints to the `servers-read` and `servers-write` capability instead? if you can "read" a server, you should probably be able to see the server's server capabilities and if you can "write" a server, you should probably be able to update the server's server capabilities....

it makes the capabiities a little less granular but that might be a good thing and i think we've done that with other endpoints...

Reply:
Ill defer to you guys! If you think that is the better path happy to do it. 

@rawlinp @rob05c  thoughts?

Reply:
I'd agree, I can't see a TC Admin wanting to ever let a person change only server caps, but nothing else on the server. Likewise, I can't imagine a DS Tenant ever wanting to let one of their employees change DS caps, but nothing else on the ds.

I think lack of granularity is fine here. That said, I don't have a strong opinion either way.  

+0

Reply:
consider it done! follow on commit coming

Reply:
+1 sounds good to me